api/protobuf-spec/backend.proto

```diff
@@ -57,16 +58,31 @@ service Backend {
   //  - error. Empty if no error was encountered
   //  - rosters, if you choose to fill them in your MMF. (Recommended)
   //  - pools, if you used the MMLogicAPI in your MMF. (Recommended, and provides stats)
-  rpc CreateMatch(CreateMatchRequest) returns (CreateMatchResponse) {} 
+  rpc CreateMatch(CreateMatchRequest) returns (CreateMatchResponse) {
+    option (google.api.http) = {
+      put: "/v1/backend/matches"
+      body: "match"
```

Comment:
Use `body: "*"`. Just to keep things consistent. It's not clear how many fields a top level request will have in the new API so for future compatibility I'd just use `body: "*"` here.